source/core/solid_base.cc

```diff
@@ -244,25 +240,21 @@ SolidBase<dim, spacedim>::setup_triangulation(const bool restart)
 
 template <>
 void
-SolidBase<2, 2>::rotate_grid(double angle, int /*axis*/)
+SolidBase<2, 2>::rotate_grid(double angle, Tensor<1, 3> /*axis*/)
```

Comment:
```suggestion
SolidBase<2, 2>::rotate_grid(const double angle, const Tensor<1, 3> /*axis*/)
```
And so on and so forth of the other ones.